scripts/cfFindIP.sh

```diff
@@ -184,16 +207,20 @@ do
 		echo "will use local file"
 		cloudFlareIpList=$(cat "$scriptDir"/cf.local.iplist)
```

Comment:
This variable is just list and is not total IP count

Reply:
You're right, but handling of shared source of threads is an extra work for this project. I just made a total progress bar that shows progress of checked main IPs and then about subnets I preferred to use Parallel's progress bar, this means we have two progress bars that first one (that's about total IPs not subnets) is handled by script and second is not handled.

Reply:
![image](https://user-images.githubusercontent.com/33148504/218172055-ee6f2f33-8684-440f-8c74-26322466d898.png)

Reply:
![image](https://user-images.githubusercontent.com/33148504/218186722-1be7c8d3-c78e-4ec5-9c0d-31e5ca3db21e.png)